src/app/profiles/profiles.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { TestBed, async } from '@angular/core/testing';
+import { ProfilesComponent } from './profiles-app.component';
+import { ProfilesModule } from './profiles-app.module';
+import { RunboxWebmailAPI } from '../rmmapi/rbwebmail';
+import { RouterTestingModule } from '@angular/router/testing';
+import { of, Observable } from 'rxjs';
+import { Profile } from './profile';
+import { LocationStrategy, APP_BASE_HREF } from '@angular/common';
+
+describe('ProfilesComponent', () => {
```

Comment:
Can we have some basic tests here? :)

Reply:
no tests for now in identities components :~